environments/pull-environment.sh

```diff
@@ -2,8 +2,12 @@
 set -e
 IFS='|'
 
+# Get args
+dir=$1
+region=$2
+
 # In development, AWS_PROFILE should be set. In CI, it's not.
-[ "$AWS_PROFILE" ] && useProfile="true" || useProfile="false";
```

Comment:
Ending line with semicolons in bash scripts are superfluous, because newline acts as command separator anyway. Removed these `;` while I was here.